crates/test-case-core/src/test_case.rs

```diff
@@ -29,34 +34,53 @@ where
     I: IntoIterator<Item = Expr>,
 {
```

Comment:
I wanted to remove this trait implementation, but it's part of the public interface and I didn't think it was worth forcing a major bump of the crate version